lib/pbench/cli/server/pbench_user_create.py

```diff
@@ -86,9 +85,6 @@ def main(context, username, password, email, first_name, last_name):
         rv = PbenchCreateUser(context).execute()
     except BadConfig as exc:
         click.echo(exc, err=True)
-        rv = 2
-    except Exception as exc:
-        click.echo(exc, err=True)
-        rv = 1
+        rv = 2 if isinstance(exc, BadConfig) else 1
```

Comment:
The `except` clause (at new line 86) needs to catch `Exception` (or something more broad than `BadConfig`).